@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -1,18 +1,25 @@
 <template>
   <li v-if="host" class="infra-host-item">
-    <InfraItemLabel
-      :current="isCurrentHost"
-      :icon="faServer"
-      :route="{ name: 'host.dashboard', params: { uuid: host.uuid } }"
+    <WithTooltip
+      :label="host.name_label"
+      v-slot="{ events }"
+      check-ellipsis-html-selector=".text"
     >
-      {{ host.name_label }}
-      <template #actions>
-        <InfraAction
-          :icon="isExpanded ? faAngleDown : faAngleUp"
-          @click="toggle()"
-        />
-      </template>
-    </InfraItemLabel>
+      <InfraItemLabel
+        v-on="events"
```

Comment:
The `WithTooltip` component cannot pass this itself? We have to pass it manually every time we want to use it?

Reply:
Not really possible.

I tried after your comment, but came across another problem that I didn't see first.

For example, if we have hundreds of hosts, this will create hundreds of Tooltip instance (and hundreds of `<!--v-if-->` at document root.

So I just rewrote the tooltip completely and I'm almost done.

Now there will be only 1 instance of the tooltip.

And instead of a WithTooltip wrapper component, i created a Directive.

The usage will be the following:

```html
<span v-tooltip="'Some help'">Here is the content</span>
```

Which seems much easier.